### PR TITLE
Add deseq2_vsn parameter

### PR DESCRIPTION
**Pull request by drpatelh**
Many thanks to contributing to nf-core/atacseq!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, also make a PR on the [nf-core/atacseq branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/atacseq)
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [x] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/atacseq/tree/master/.github/CONTRIBUTING.md

This just allows users to pick the much faster `vsn` transform over `rlog` for differential analysis

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,7 +14,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * [#79](https://github.com/nf-core/atacseq/issues/79) - Parallelize DESeq2
 * [#80](https://github.com/nf-core/atacseq/pull/80) - Added social preview image
 * Update template to tools `1.9`
-* Parameter `--skip_consensus_peaks` to skip consensus peak generation
+* Parameters:
+  * `--skip_consensus_peaks` to skip consensus peak generation
+  * `--deseq2_vst` to use `vst` transformation instead of `rlog` with DESeq2
```
Comment (mashehu) [inline CHANGELOG.md]: Not sure how strict you are, but technically `vst` already has transformation in its name, maybe you can use 'normalization' instead. Same for other occurrences later on.

File: docs/output.md [hunk 2]
```diff
@@ -210,7 +210,7 @@ The library-level alignments associated with the same sample are merged and subs
     *Description*:  
     DESeq2 is more commonly used to perform differential expression analysis for RNA-seq datasets. However, it can also be used for ATAC-seq differential accessibility analysis, in which case you can imagine that instead of counts per gene for RNA-seq data we now have counts per accessible region.  
 
-    This pipeline uses a standardised DESeq2 analysis script to get an idea of the reproducibility within the experiment, and to assess the overall differential binding. Please note that this will not suit every experimental design, and if there are other problems with the experiment then it may not work as well as expected.
+    This pipeline uses a standardised DESeq2 analysis script to get an idea of the reproducibility within the experiment, and to assess the overall differential binding. Please note that this will not suit every experimental design, and if there are other problems with the experiment then it may not work as well as expected. For larger experiments, it may be recommended to use the `vst` transformation instead of the default `rlog` option. You can do this by providing the `--deseq2_vst` parameter to the pipeline.
```
Comment (mashehu) [inline docs/output.md]: I would quickly clarify why you recommend it for larger experiments.

File: CHANGELOG.md (hunk 1 above)
Comment (drpatelh) [reply CHANGELOG.md]: Fixed https://github.com/nf-core/atacseq/pull/93/commits/a1516de9689401afd84e5a2617d08ba539e9765f

File: docs/output.md (hunk 2 above)
Comment (drpatelh) [reply docs/output.md]: Good spot with the typo!

Comment (drpatelh) [reply docs/output.md]: Easier to link to the docs! Fixed https://github.com/nf-core/atacseq/pull/93/commits/391ae6ad9b277d8f3b26f3c944e2059bcac0c7a5

**Comment by drpatelh** [comment]
Thanks alot @mashehu . Would you mind approving this PR too because they are pretty much the same changes:
https://github.com/nf-core/chipseq/pull/145

**Pull request merged**